website/sidebars.js

```diff
@@ -341,5 +341,22 @@ module.exports = {
                 },
             ],
         },
+        {
+            label: 'Contributing',
```

Comment:
Can we make this clearer?  Something like this?

```suggestion
            label: 'Developer contribution guides',
```